example/src/Scatter.purs

```diff
@@ -93,11 +94,17 @@ options sinData cosData = do
 
 chart ∷ ∀ e. Eff (dom ∷ DOM, echarts ∷ ET.ECHARTS, err ∷ EXCEPTION, random ∷ RANDOM|e) Unit
 chart = do
-  mbEl ← U.getElementById $ ElementId "scatter"
+  chart' (ElementId "scatter-1") Nothing

```

Comment:
That's not about examples, but about API. If we preserve `chart` signature, then API isn't changed and it's a minor version bump. 
I've just put comment here, not there :D